flags/src/main/java/com/yahoo/vespa/flags/Flags.java

```diff
@@ -157,6 +158,13 @@
             "Takes effect on deployment through controller",
             APPLICATION_ID);
 
+    public static final UnboundBooleanFlag DISABLE_CHEF = defineFeatureFlag(
+            "disable-chef", false,
+            "Stops and disables chef-client",
+            "Takes effect on next host-admin tick",
+            ZONE_ID, HOSTNAME);
```

Comment:
Do not use `ZONE_ID` for flags unless its for controller, see: https://github.com/vespa-engine/vespa/blob/957bbe9989a63a6e930a05b2e1f596b10a0c31b5/flags/src/main/java/com/yahoo/vespa/flags/FetchVector.java#L28